src/pages/docs/liveobjects/map.mdx

```diff
@@ -94,8 +122,30 @@ const { unsubscribe } = map.subscribe(() => console.log('Map updated'));
 // To remove the listener
 unsubscribe();
 ```
+
+```swift
+// Initial subscription
+let subscriptionResponse = try map.subscribe { _, _ in
+    print("Map updated")
+}
+// To remove the listener
+subscriptionResponse.unsubscribe()
+```
 </Code>
 
+<If lang="swift">
+To remove a map update listener from _inside_ the listener function, you can call `unsubscribe()` on the subscription response that is passed as the second argument to the listener function:
+
+```swift
```

Comment:
This needs to be wrapped in `<Code>` tags.

Reply:
Thanks, done